yew-functional/src/lib.rs

```diff
@@ -61,7 +56,7 @@ pub trait FunctionProvider {
 }
 
 #[derive(Clone, Default)]
-struct MsgQueue(Rc<RefCell<Vec<Msg>>>);
+pub struct MsgQueue(Rc<RefCell<Vec<Msg>>>);
```

Comment:
What's the reason for making this public?

Reply:
None, my bad.